runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -677,6 +680,40 @@ private void processInventoryLoot(String event, LootRecordType lootRecordType, I
 		}
 	}
 
+	private boolean processHerbSackLoot(int timestamp, String event, LootRecordType lootRecordType)
+	{
+		List<ItemStack> herbs = new ArrayList<>();
+		for (Object messageObject : client.getMessages())
+		{
+			MessageNode messageNode = (MessageNode) messageObject;
+			if (messageNode.getTimestamp() == timestamp && messageNode.getType() == ChatMessageType.SPAM)
```

Comment:
This would be cleaner as a guard clause.
```suggestion
			if (messageNode.getTimestamp() != timestamp || messageNode.getType() != ChatMessageType.SPAM)
			{
				continue;
			}
```